eZ Platform/APIs/REST/REST_v3.md

```diff
@@ -55,87 +62,73 @@ Fetching an object by location. Location can be either a location ID (42) or a r
 
 ## Example: fetching an object
 
-```
-Resource: /content/object/(location_id)
-Method: GET
+__Resource__: _/content/object/<LOCATION_ID>_
```

Comment:
Afaik the requriment is to be able to get content by location id.

@bdunogier We can make similar entrypoint like for [remote id](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rest/REST-API-V2.rst#load-content-by-remote-id) here right?

So:
```
GET /content/objects?locationId=<LOCATION_ID>

HTTP/1.1 307 Temporary Redirect
Location: /content/objects/<id>
```